tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode.xmi

```diff
@@ -89,7 +89,7 @@
     </commands>
     <commands name="EndScan" description="" execMethod="end_scan" displayLevel="OPERATOR" polledPeriod="0" isDynamic="false">
       <argin description="">
-        <type xsi:type="pogoDsl:StringType"/>
+        <type xsi:type="pogoDsl:VoidType"/>
```

Comment:
It is okay to take no input argument in EndScan. However, going forward - there will a scenario for timestamped command. So one argument will be there at least - a timestamp at which EndScan shall invoke.

Reply:
Nice suggestion. It will be changed whenever required.